sites/diverseeducation.com/server/components/nodes/arthur-ashe-profiles.marko

```diff
@@ -0,0 +1,60 @@
+import { get, getAsObject, getAsArray } from "@parameter1/base-cms-object-path";
+import defaultValue from "@parameter1/base-cms-marko-core/utils/default-value";
+
+$ const content = getAsObject(input, "node");
+$ const profile = getAsArray(content, "profiles")[0];
```

Comment:
Loop over all profiles.

Reply:
I left this as it is on the given node and is used for the name and body being displayed on the page